python/pyspark/pandas/missing/window.py

```diff
@@ -124,3 +142,16 @@ class MissingPandasLikeRollingGroupby:
     is_datetimelike = _unsupported_property_rolling("is_datetimelike")
     is_freq_type = _unsupported_property_rolling("is_freq_type")
     ndim = _unsupported_property_rolling("ndim")
+
+
+class MissingPandasLikeExponentialMoving:
```

Comment:
We will have to test it at `python/pyspark/pandas/tests/test_window.py`.

Reply:
Maybe you could refer to https://github.com/apache/spark/pull/36034/files when adding the missing test.